azure-pipelines/publishing/templates/release-docker-build-steps.yml

```diff
@@ -0,0 +1,74 @@
+# Template: Steps to build all components for a specific OS
+# Consume this steps template in one or more jobs by passing in parameter values.
+
+parameters:
+- name: targetOsArch        # example: debian9_arm32
+  type: string
+- name: imageVersion
+  type: string
+
+
+jobs:
+- job: ${{parameters.targetOsArch}}
+  steps:
+  - checkout: self
+    path: 's' # Copy to sources directory (templates do not do this automatically)
+  - template: ../../build/linux/du/templates/doclient-lite-docker-steps.yml
```

Comment:
>template [](http://example.com/codeflow?start=4&length=8)

Please add a blank line between template/task/job lines.

Reply:
In all yamls here.

Reply:
sure, I kept them without the line to make it more obvious that this task is related to the previous job

Reply:
Thanks. Prefer the blank line. What about lines 16, 60, 66, 73?

Reply:
Oh gotcha, misunderstood the ask - thought you meant only adding spaces between jobs and tasks. Added the additional spaces